aten/src/ATen/core/TensorBase.h

```diff
@@ -88,7 +88,7 @@ class TORCH_API TensorBase {
   // taken to avoid decrementing this reference count at destruction
   // time. Intended to support MaybeOwnedTraits<Tensor>.
   explicit TensorBase(unsafe_borrow_t, const TensorBase& rhs)
-      : impl_(c10::intrusive_ptr<at::TensorImpl, UndefinedTensorImpl>::reclaim(rhs.impl_.get())) {}
+      : impl_(c10::intrusive_ptr<at::TensorImpl, UndefinedTensorImpl>(rhs.impl_.get(), c10::raw::DontIncreaseRefcount{})) {}
```

Comment:
@cyyever This doesn't resolve the slicing issues in your PR correct?

Reply:
@Skylion007 May not be the same issue, but here some tests fail.